README.md

```diff
@@ -615,6 +615,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 
 ## Money, Budgeting and Management
 
+  * [Anchor](http://theanchorapp.com/) - Anchor is invoicing software that you host on your own server and is free to use forever ([Demo](http://theanchorapp.com/demo/admin-login)), `PHP`
```

Comment:
1. There's no source code
2. > This software can not be redistributed in any form.

Please move this to non-free.md

There's also an unneeded , at the end.